lgsm/config-default/config-lgsm/sfserver/_default.cfg

```diff
@@ -16,7 +16,7 @@ port="7777"
 
 ## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
 ## Game Server Docs | https://satisfactory.fandom.com/wiki/Dedicated_servers
-startparameters="FactoryGame -MultiHome=${ip} -Port=${port} -ServerQueryPort=${queryport} -BeaconPort=${beaconport} -log"
+startparameters="FactoryGame -Port=${port} -ServerQueryPort=${queryport} -BeaconPort=${beaconport} -log"
```

Comment:
Why has multihome been removed? This is a standard parameter that is added to all Unreal engine servers

Reply:
Please see #4644 where @ewlidd reported the following:
> Satisfactory has a new port requirement of TCP 7777. This will give an error when browsing to the server of "failed to connect to the server api"
you must edit your config and remove -multihome ${ip} from the launch flags.
for some reason, when the satisfactory launches with the multihome flag it binds to 127.0.0.1:7777 instead of the multihome IP (or 0.0.0.0).